plugins/custom-cols.yaml

```diff
@@ -0,0 +1,36 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: custom-cols
+spec:
+  version: "v0.0.2"
+  platforms:
+  - selector:
+      matchExpressions:
+      - {key: os, operator: In, values: [darwin, linux]}
+    uri: https://github.com/webofmars/kubectl-custom-cols/archive/v0.0.2.tar.gz
+    sha256: "f01d1aa63c320989071baa88d7b0a8fefa1147e0da8b9edf32874def21c84736"
+    files:
+    - from: "./kubectl-custom-cols-0.0.2/kubectl-custom-cols"
+      to: "."
+    - from: "./kubectl-custom-cols-0.0.2/templates/*.tpl"
+      to: "templates"
+    bin: "./kubectl-custom-cols"
+  shortDescription: Easy use of custom columns output formating.
+  homepage: https://github.com/webofmars/kubectl-custom-cols
+  caveats: |
+    For now you can't list witch templates are available and adding some can only
```

Comment:
off-topic: consider adding a path like ~/.kube-custom-cols that your tool recognizes, or an env variable.

Reply:
The plugin docs mention this:

> You can add you own templates in $HOME/.kube/kube-custom-cols/templates folder.

Although I think this is not working at the moment. @fredleger what's the status there?

Reply:
**off-topic:** I'd not add anything serious under ~/.kube, people delete it all the time as it contains kubeconfig that can usually be restored very easily.

we should not block on these stuff.

Reply:
If i understood correctly (and maybe i did not) how krew is working we can't copy anything outside the store/<id> folder right ?

Also i would really like to bundle the templates file with the plugin actually i do think that most of the interest in the plugin is to have a descent collection of predefined templates to use with it (and ones with real use case usage).

If it's possible to copy file outside the store folder my best choice will be `$HOME/.kube-custom-cols` or similar. I agree with @ahmetb about the possible deletion of `.kube` here.

Reply:
I think it's fine to bundle the column configuration with the plugin. You should only keep in mind that this is not a safe place to store user data, because these directories get wiped when a new version of the plugin is installed.

Reply:
I tested a bit and it seems my assumption was correct.

My idea is the following now:
- keep the templates in the krew store folder for distributing the plugin with the community templates
- in next version add the ability to search the templates in a list of folders (basically the one bundled and the users overrides/adds in `$HOME/.kube-custom-cols`.

What do you think ?